src/modules/variables.js

```diff
@@ -0,0 +1,17 @@
+export const canvas = document.getElementById('canvas');
+export const ctx = canvas.getContext('2d');
+export const mainIndent = 50;
```

Comment:
Any reason for export const `mainIndent`. Looks like you use this variable only inside this file?